text_generation/causal_lm/cpp/requirements.txt

```diff
@@ -1,5 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
-optimum[openvino]==1.19.1
+optimum[openvino]==1.19.2
```

Comment:
@Wovchena maybe we need to use `optimum-intel[openvino]` instead?
For us, changes in `optimum` maybe don't have too much value